thorclient/common/common.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2024 The VeChainThor developers
+
+// Distributed under the GNU Lesser General Public License v3.0 software license, see the accompanying
+// file LICENSE or <https://www.gnu.org/licenses/lgpl-3.0.html>
+
+package common
```

Comment:
Does this need its own package? Seems tiny and could easily be placed inline with httpclient or thorclient

Reply:
It's imported by multiple packages. Errors for error testing, `TxSendResult` for tx results and the EventWrapper for other uses.

But it's a good point, what if I leave this comment open and review it again once the PR is closer to done ?